src/js/articleViewToolbar.js

```diff
@@ -87,55 +81,62 @@ function addTag(itemId, tagName) {
 }
 */
 
-function render(itemId) {
-  chrome.runtime.sendMessage(null, {command:'getBookmark', id: itemId}, function(item) {
-
-    document.getElementById('menuDelete').onclick = function() {
-      deleteItemRequest(itemId);
-      this.className="deleted";
-    };
-
-    var menuToggleStar = document.getElementById('menuToggleStar');
-    menuToggleStar.innerHTML = '';
-    var star = document.createElement('i');
-    star.className = 'icon ion-android-star';
-    menuToggleStar.className = item.favorite ? 'star-on' : 'star-off';
-    menuToggleStar.appendChild(star);
-    menuToggleStar.onclick = function() {
-      if (menuToggleStar.className === 'star-on') {
-        removeStarRequest(itemId);
-        menuToggleStar.className = 'star-off';
-      } else {
-        addStarRequest(itemId);
-        menuToggleStar.className = 'star-on';
-      }
-    };
-
-    document.getElementById('menuArchive').onclick = function() {
-      archiveItemRequest(itemId);
-      this.className="archived";
-    };
-
-    /*
-    _.forEach(item.tags, function(tagName) {
-      var deleteTagId = '#delete_' + getTagId(tagName);
-      $('#menuTags').append(generateTagHtml(tagName));
-      $(deleteTagId).click(function() {
-        deleteTag(itemId, tagName);
-      });
-    });
+function render(article) {
+  let itemId = article.item_id;
+
+  document.getElementById('menuDelete').onclick = function() {
+    deleteItemRequest(itemId);
+    this.className="deleted";
+  };
 
-    $('#addTagForm').submit(function(e) {
-      e.preventDefault();
-      var newTagName = $('#menuAddTag').val();
-      addTag(itemId, newTagName);
-      $('#menuAddTag').val('');
+  var menuToggleStar = document.getElementById('menuToggleStar');
+  menuToggleStar.innerHTML = '';
+  var star = document.createElement('i');
+  star.className = 'icon ion-android-star';
+  menuToggleStar.className = article.favorite ? 'star-on' : 'star-off';
+  menuToggleStar.appendChild(star);
+  menuToggleStar.onclick = function() {
+    if (menuToggleStar.className === 'star-on') {
+      removeStarRequest(itemId);
+      menuToggleStar.className = 'star-off';
+    } else {
+      addStarRequest(itemId);
+      menuToggleStar.className = 'star-on';
+    }
+  };
+
+  let status = parseInt(article.status);
+  if (status === 1) {
+    document.getElementById('menuArchive').className = 'archived';
+  }
+  if (status === 2) {
+    document.getElementById('menuDelete').className = 'deleted';
+  }
+
+  document.getElementById('menuArchive').onclick = function() {
+    archiveItemRequest(itemId);
+    this.className="archived";
+  };
+
+  /*
+  _.forEach(article.tags, function(tagName) {
+    var deleteTagId = '#delete_' + getTagId(tagName);
+    $('#menuTags').append(generateTagHtml(tagName));
+    $(deleteTagId).click(function() {
+      deleteTag(itemId, tagName);
     });
-    */
+  });
 
+  $('#addTagForm').submit(function(e) {
```

Comment:
those lines are commented because in the end we agreed upon the basic version of the article view panel (without tagging). But, if we eventually want to make that functionality, it will be easier with this prepared code.